packages/amplication-client/src/Workspaces/WorkspaceHeader/WorkspaceHeader.tsx

```diff
@@ -52,7 +52,7 @@ const WorkspaceHeader: React.FC<{}> = () => {
       (isResourceRoute && currentResource) ||
       (isResourceRoute && currentProjectConfiguration)
     )
-      return currentResource?.name || currentProjectConfiguration?.name;
+      return currentResource?.resourceType ===  models.EnumResourceType.Service ?  currentResource?.name : 'Project Configuration'
```

Comment:
use the condition based on ProjectConfiguration since there are other resource types

```suggestion
      return currentResource?.resourceType ===  models.EnumResourceType.ProjectConfiguration ?  'Project Configuration' : currentResource?.name
```

Reply:
Done